### PR TITLE
Remove \GetDocumentCommandArgSpec, etc. from base

### PR DESCRIPTION
**Pull request by josephwright**
# Internal housekeeping

## Status of pull request

- [x] Ready to merge

## Checklist of required changes before merge will be approved
- [ ] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [x] `ltnewsX.tex` (and/or `latexchanges.tex`) updated

### CODE REVIEW

File: base/changes.txt [hunk 1]
```diff
@@ -6,6 +6,11 @@ completeness or accuracy and it contains some references to files that
 are not part of the distribution.
 ================================================================================
 
+2023-08-19 Joseph Wright  <Joseph.Wright@latex-project.org>
+
+	* ltcmd.dtx, usrguide.tex
+	Remove \GetDocumentCommandArgSpec, etc.
```
Comment (muzimuzhi) [inline base/changes.txt:12]: Guessing Frank may ask to list them all.

**Comment by muzimuzhi** [comment]
Need to run
```
l3build save -c config-ltcmd -e luatex ltcmd004 ltcmd005
```
to update luatex specific tlg files for tests `ltcmd004` and `ltcmd005`.

File: base/changes.txt (hunk 1 above)
Comment (josephwright) [reply base/changes.txt]: Done

Comment (FrankMittelbach) [reply base/changes.txt]: > Guessing Frank may ask to list them all.

:-) but yes, if I had noticed

**Review by FrankMittelbach — APPROVED** [review]

**Pull request merged**